hkmc2/shared/src/main/scala/hkmc2/codegen/HandlerLowering.scala

```diff
@@ -279,11 +280,10 @@ class HandlerLowering(using TL, Raise, Elaborator.State, Elaborator.Ctx):
   
   private def translateBlock(b: Block, h: HandlerCtx): Block =
     given HandlerCtx = h
+    
```

Comment:
It's weird to have an empty lie here in the middle of `translateBlock`'s body but none between `translateBlock` and `firstPass`.